bigbluebutton-html5/imports/ui/components/external-video-player/external-video-player-graphql/component.tsx

```diff
@@ -248,7 +248,8 @@ const ExternalVideoPlayer: React.FC<ExternalVideoPlayerProps> = ({
 
   useEffect(() => {
     if (playerRef.current) {
-      playerRef.current.seekTo(currentTime, 'seconds');
+      const truncatedTime = currentTime < 1 ? 0 : currentTime;
```

Comment:
Isn't it better to always use `const truncatedTime = Math.floor(currentTime);`?
Once it is type `number`, it's better to round every time, even when it's higher than 1!

Reply:
Yeah, I tested it this way. Even though this would be the more ellegant solution, this causes slight backward jumps when playing/pausing. Since our issue is when `currentTime` is between 0 and 1, I opted to truncate when < 1 and preserving the original fractional number otherwise.

Reply:
Sounds  good!